README.md

```diff
@@ -11,60 +13,77 @@ vSphere data is obtained via a client provided by the `govmomi` package, the off
 
 ## Installation
 
-To install the integration, follow the official [documentation](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/vmware-vsphere-monitoring-integration). We recommend using a package manager.
+To install the integration, follow the official [documentation](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/vmware-vsphere-monitoring-integration). We recommend using your operating system native package manager.
+
+## Getting started
+
+After you've [installed](#installation) the integration you should make sure that the required configuration is appropriate for your environment.
```

Comment:
```suggestion
After you've [installed](#installation) the integration make sure that you've the required configuration for your environment.
```

Reply:
hum.. not sure the contraction here reads well.. you mean "...that you have the..." right? I would not contract to "you've" then, reads a bit weird, no?